README.md

```diff
@@ -174,38 +177,48 @@ For larger cohorts, samples should be split up into batches of about 100-500 sam
 
 ## <a name="gcnv-training-overview">gCNV Training</a>
 Both the cohort and single-sample modes use the GATK gCNV depth calling pipeline, which requires a [trained model](#gcnv-training) as input. The samples used for training should be technically homogeneous and similar to the samples to be processed (i.e. same sample type, library prep protocol, sequencer, sequencing center, etc.). The samples to be processed may comprise all or a subset of the training set. For small, relatively homogenous cohorts, a single gCNV model is usually sufficient. If a cohort contains multiple data sources, we recommend training a separate model for each [batch](#batching) or group of batches with similar dosage score (WGD). The model may be trained on all or a subset of the samples to which it will be applied; a reasonable default is 100 randomly-selected samples from the batch (the random selection can be done as part of the workflow by specifying a number of samples to the `n_samples_subsample` input parameter in `/wdl/TrainGCNV.wdl`).
 
+## <a name="reference-panel-generation">Generating a reference panel</a>
+New reference panels can be generated easily from a single run of the `GATKSVPipelineBatch` workflow. If using a Cromwell server, we recommend copying the outputs to a permanent location by adding the following option to the workflow configuration file:
+```
+  "final_workflow_outputs_dir" : "gs://my-outputs-bucket",
+  "use_relative_output_paths": false,
+```
+Here is an example of how to generate workflow input jsons from `GATKSVPipelineBatch` workflow metadata:
+```
+> cromshell -t60 metadata 38c65ca4-2a07-4805-86b6-214696075fef > metadata.json
+> python scripts/inputs/create_test_batch.py \
+    --execution-bucket gs://my-exec-bucket \
+    --final-workflow-outputs-dir gs://my-outputs-bucket \
+    metadata.json \
+    > inputs/values/my_ref_panel.json
+> # Build test files for batched workflows
+> python scripts/inputs/build_inputs.py \
+    inputs/values \
+    inputs/templates/test \
+    inputs/build/my_ref_panel/test \
+    -a '{ "test_batch" : "ref_panel_1kg" }'
+> # Build test files for the single-sample workflow
+> python scripts/inputs/build_inputs.py \
+    inputs/values \
+    inputs/templates/test/GATKSVPipelineSingleSample \
+    inputs/build/NA19240/test_my_ref_panel \
+    -a '{ "single_sample" : "test_single_sample_NA19240", "ref_panel" : "my_ref_panel" }'
+> # Build files for a Terra workspace
+> python scripts/inputs/build_inputs.py \
+    inputs/values \
+    inputs/templates/terra_workspaces/single_sample \
+    inputs/build/NA12878/terra_my_ref_panel \
+    -a '{ "single_sample" : "test_single_sample_NA12878", "ref_panel" : "my_ref_panel" }'
+```
```

Comment:
Could also note that build_default_inputs.py can be used to build all at once

Reply:
Well that has ref_panel_1kg hard-coded, so you couldn't use it without modifying it?